cmake/modules/SwiftSource.cmake

```diff
@@ -348,7 +348,7 @@ function(_compile_swift_files
     list(APPEND module_outputs "${interface_file}")
   endif()
 
-  swift_install_in_component("${SWIFTFILE_INSTALL_IN_COMPONENT}"
+  swift_install_in_component("${SWIFTFILE_INSTALL_IN_COMPONENT}" ${module_base}-module
```

Comment:
I think that it would be better to install the module with the module itself.  You cannot use the module for development otherwise.